src/ui.form-calendar/useAttrs.js

```diff
@@ -1,375 +1,179 @@
+      base: computed(() => [
+        extendingDayKeyClasses.day.value,
+        extendingKeysClasses.anotherMonthDate.value,
+      ]),
+    },
+    currentDay: {
+      base: computed(() => [
+        extendingDayKeyClasses.day.value,
+        extendingKeysClasses.currentDate.value,
+      ]),
+    },
+    currentDayInRange: {
+      base: computed(() => [
+        extendingDayKeyClasses.day.value,
+        extendingKeysClasses.dateInRange.value,
+        extendingKeysClasses.currentDate.value,
+      ]),
+    },
+    firstDayInRange: {
+      base: computed(() => [
+        extendingDayKeyClasses.day.value,
+        extendingKeysClasses.edgeDateInRange.value,
+        extendingKeysClasses.firstDateInRange.value,
+      ]),
+    },
+    lastDayInRange: {
+      base: computed(() => [
+        extendingDayKeyClasses.day.value,
+        extendingKeysClasses.edgeDateInRange.value,
+        extendingKeysClasses.lastDateInRange.value,
+      ]),
+    },
+    anotherMonthLastDayInRange: {
+      base: computed(() => [
+        extendingDayKeyClasses.day.value,
+        extendingKeysClasses.anotherMonthDay.value,
+        extendingKeysClasses.edgeDateInRange.value,
+        extendingKeysClasses.lastDateInRange.value,
+      ]),
+    },
+    anotherMonthFirstDayInRange: {
+      base: computed(() => [
+        extendingDayKeyClasses.day.value,
+        extendingKeysClasses.anotherMonthDay.value,
+        extendingKeysClasses.edgeDateInRange.value,
+        extendingKeysClasses.firstDateInRange.value,
+      ]),
+    },
+    currentMonth: {
+      base: computed(() => [extendingKeysClasses.currentDate.value]),
+    },
+    monthInRange: {
+      base: computed(() => [extendingKeysClasses.dateInRange.value]),
+    },
+    singleMonthInRange: {
+      base: computed(() => [extendingKeysClasses.dateInRange.value]),
+    },
+    currentMonthInRange: {
+      extend: computed(() => [
+        extendingKeysClasses.currentDate.value,
+        extendingKeysClasses.dateInRange.value,
+      ]),
+    },
+    lastMonthInRange: {
+      extend: computed(() => [
+        extendingKeysClasses.edgeDateInRange.value,
+        extendingKeysClasses.lastDateInRange.value,
+      ]),
+    },
+    firstMonthInRange: {
+      extend: computed(() => [
+        extendingKeysClasses.edgeDateInRange.value,
+        extendingKeysClasses.firstDateInRange.value,
+      ]),
+    },
+    selectedMonth: {
+      base: computed(() => [extendingKeysClasses.selectedDate.value]),
+    },
+    activeMonth: {
+      base: computed(() => [extendingKeysClasses.activeDate.value]),
+    },
+    currentYear: {
+      base: computed(() => [extendingKeysClasses.currentDate.value]),
+    },
+    yearInRange: {
+      base: computed(() => [extendingKeysClasses.dateInRange.value]),
+    },
+    singleYearInRange: {
+      base: computed(() => [extendingKeysClasses.dateInRange.value]),
+    },
+    currentYearInRange: {
+      extend: computed(() => [
+        extendingKeysClasses.currentDate.value,
```

Comment:
It looks like the logic is wrong:
![image](https://github.com/user-attachments/assets/2aa90d2f-96f1-46e0-83f2-94e4e514ad86)

Double check all configs again. Here is non typical logic (was applied before), be aware to it.

Reply:
I've checked and changed extendingKeys strategy from `extend` to `base`, add separate `"month"` and `"year" `keys to `getExtendingKeysClasses` function